chore: Update docs to add universal RPC info (#1156)
Unity-Technologies/com.unity.multiplayer.docs: docs/tutorials/get-started-with-ngo.md

If the player is a server-owned player at `OnNetworkSpawn()`, you can immediately move this player, as suggested in the following code.

```csharp
            SubmitPositionRequestServerRpc();
```

> This `Rpc` can be called regardless of whether the player is a client or a server. When an `Rpc` with `SendTo.Server` is called on the server side, it simply executes as if it were a local function call by default.

Reviewer: ```suggestion
You can call this `Rpc` when the player is a client or a server. When you call an `Rpc` with `SendTo.Server`  on the server side. it executes in the same way as a local function call by default.
```
